datahub-web-react/src/conf/theme/theme_dark.config.json

```diff
@@ -27,6 +27,20 @@
         },
         "search": {
             "searchbarMessage": "Search Datasets, People, & more..."
+        },
+        "menu": {
+            "items": [
+                {
+                    "label": "Datahub Wiki",
+                    "path": "https://github.com/linkedin/datahub",
+                    "shouldOpenInNewTab": true
+                },
+                {
+                    "label": "See metadata origin",
+                    "path": "http://go/metadata/origins",
+                    "shouldOpenInNewTab": true
```

Comment:
I don't know if it makes sense to include this as a default menu item, since it won't work for anyone in the community. Maybe leave this out and allow Linkedin to add it back in on their fork?

Reply:
Yes, you are right. It don't make sense to put this as a default menu item, so I have removed it.